test/fixtures/fulfillment_order.json

```diff
@@ -0,0 +1,41 @@
+{
+   "fulfillment_order": {
+      "id": 519788021,
+      "shop_id": 690933842,
+      "order_id": 450789469,
+      "fulfillment_service_handle": "shipwire",
+      "status": "open",
+      "requires_shipping": true,
+      "is_not_deleted": true,
+      "deleted_at": null,
+      "created_at": "2016-07-12T11:23:42-04:00",
+      "updated_at": "2016-07-12T11:23:42-04:00",
+      "happened_at": "2016-07-12T11:23:42-04:00",
+      "assigned_location_id": 905684977,
+      "request_status": "unsubmitted",
+      "delivery_category": null,
+      "fulfillment_order_line_items": [
+         {
+            "id": 519788021,
+            "shop_id": 690933842,
+            "fulfillment_order_id": 519788021,
+            "line_item_id": 466157049,
+            "variant_id": 43729076,
+            "inventory_item_id": 808950810,
+            "inventory_commitment_id": null,
```

Comment:
Still missing an `inventory_commitment_id` here. Since these fixtures seem to be completely isolated, I think the foreign key id values can be anything.

Reply:
I thought that `inventory_commitment_id` were nullable and should be filled in when it made sense to do so rather than always having a value to clear out when not needed. Is that not the case?

Reply:
You're right that a fulfillment order line item may not always have a corresponding commitment (like when a variant is deleted, maybe digital products in the future), but that's more of an edge case.